src/App.js

```diff
@@ -0,0 +1,19 @@
+/** @format */
+
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import HomePage from './pages/Home';
+import Categories from './pages/Categories';
+import './App.css';
+
+function App() {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<HomePage />} />
+        <Route path="/categories" element={<Categories />} />
+      </Routes>
+    </BrowserRouter>
+  );
+}
+
```

Comment:
[OPTIONAL]
- [ ] Nice work implementing the navigation bar :clap: However, It would be nice to render the `Navbar.js` component in the `App.js` component to avoid repeating the same code.